new-components/pkg/target/target.go

```diff
@@ -0,0 +1,9 @@
+package target
```

Comment:
an enum listing available target, could be used by validation to ensure we don't use random targets

Reply:
I think that this is already available https://github.com/smithy-security/smithy/blob/main/proto/ocsf/ocsf_ext/finding_info/v1/finding_info.proto and can be imported from the local generated code in the SDK